experimental/FTheoryTools/docs/src/introduction.md

```diff
@@ -76,13 +81,14 @@ Future extensions include, but are not necessarily limited to, the following:
 
 Please direct questions about this part of OSCAR to the following people:
 * [Martin Bies](https://martinbies.github.io/),
-* [Andrew Turner](https://apturner.net/).
+* [Andrew Turner](https://apturner.net/),
+* [Mikelis Emils Mikelsons](https://github.com/emikelsons).
```

Comment:
I would suggest alphabetic order by last name. Unless I am mistaken, this is the accepted standard OSCAR-wide.

Reply:
So Bies, Mikelsons, Turner.